examples/sam/template.yml

```diff
@@ -1,14 +1,14 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
-Description: "IOPipe PyPy 3.5 Runtime SAM Example"
+Description: "PyPy Runtime SAM Example"
 
 Globals:
   Function:
     Layers:
-        - arn:aws:lambda:us-east-1:146318645305:layer:pypy35:1
+        - arn:aws:lambda:us-east-1:146318645305:layer:pypy35600:1
```

Comment:
Where does this ARN come from?

Reply:
It's the iopipe account id i suspect
(which is the original repo this project was forked from)